cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/areainterchange/CgmesControlArea.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.elements.areainterchange;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+public class CgmesControlArea {
+    private final String id;
+    private final String name;
+    private final String energyIdentCodeEic;
+    private final Set<EquipmentEnd> equipmentEnds = new HashSet<>();
+    private final double netInterchange;
+
+    public CgmesControlArea(String id, String name, String energyIdentCodeEic, double netInterchange) {
+        this.id = id;
+        this.name = name;
+        this.energyIdentCodeEic = energyIdentCodeEic;
+        this.netInterchange = netInterchange;
+    }
+
+    public void addTerminal(String equipmentId, int end) {
+        equipmentEnds.add(new EquipmentEnd(equipmentId, end));
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getEnergyIdentCodeEic() {
+        return energyIdentCodeEic;
+    }
+
+    public Set<EquipmentEnd> getTerminals() {
+        return equipmentEnds;
+    }
+
+    public double getNetInterchange() {
+        return netInterchange;
+    }
+
+    public static class EquipmentEnd {
```

Comment:
By end you mean side?

Reply:
Yes, I use the same nomenclature used in PR#1540 by Miora